### PR TITLE
Update previous context tag constant for person name

### PR DESCRIPTION
**Pull request by tanaya-b**
## JIRA Ticket Number

JIRA TICKET: SUPPORT-2597

## Description of change
Remove singular "name" tags from multiple languages to avoid false entity capturing.

## Checklist (OPTIONAL):

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: ner_v1/detectors/textual/name/lang_constants.py [hunk 1]
```diff
@@ -636,21 +636,21 @@
...
-    MALAY_LANG: ["nama", "*nama*", "nama awak", "alamat awak", "panggil awak", "ketahui namamu", "nama penuh anda",
+    MALAY_LANG: ["nama awak", "alamat awak", "panggil awak", "ketahui namamu", "nama penuh anda",
                  "sebutkan namamu", "siapa nama awak", "Adakah nama awak", "kongsi nama anda", "berikan nama anda",
                  "beritahu nama anda", "masukkan nama anda", "sebutkan nama anda", "Apakah nama penuh anda?",
                  "beritahu saya nama anda", "lupa menyebut nama anda", "sila masukkan nama anda",
                  "apa yang harus anda hubungi", "bantu saya dengan nama awak", "masukkan nama penuh anda lagi",
                  "bantu saya dengan nama penuh anda", "tolong beritahu saya nama penuh anda.",
                  "Sepertinya anda lupa menyebut nama anda", "tolong bantu saya dengan nama penuh anda"],
-    BAHASA_LANG: ["nama", "nama", "namamu", "tahu namamu", "siapa namamu", "adalah namamu", "Menelepon kamu",
+    BAHASA_LANG: ["nama","namamu", "tahu namamu", "siapa namamu", "adalah namamu", "Menelepon kamu",
```
Comment (haptik-deployment) [inline ner_v1/detectors/textual/name/lang_constants.py:646]: E231 missing whitespace after ','

File: ner_v1/detectors/textual/name/lang_constants.py [hunk 2]
```diff
@@ -840,7 +840,8 @@
                   "مرحبأ، أنا إرغو، المساعد الإلكتروني لغودريج بروبرتيز و أنا هنا لمساعدتك كي تجد عقارك المثالي. هل يمكنني معرفة إسمك؟",
                   "لقد أتيت إلى المكان المناسب. أستطيع مساعدتك للحصول على برنامج الإئتمان الذي يناسب إحتياجاتك! لكن اولأ، أرغب بمعرفتك بشكل أفضل.ما إسمك؟",
                   "مرحبأ، أنا إرغو، المساعد الإلكتروني لغودريج بروبرتيز و انا هنا لأعرض عليك أفضل المساحات التجارية لتلبية طلباتك . هل يمكنني معرفة إسمك؟"],
-    GERMAN_LANG: ["name", "heißt", "namen", "heißen", "Wie heißt du?", "Ihren Namen, bitte", "Bitte Name eingeben",
+    GERMAN_LANG: ["dein Name", "Ihr Name", "heißt", "namen", "heißen", "Wie heißt du?", "Ihren Namen, bitte",
+                  "Bitte Name ","eingeben",
```
Comment (haptik-deployment) [inline ner_v1/detectors/textual/name/lang_constants.py:844]: E231 missing whitespace after ','

File: ner_v1/detectors/textual/name/lang_constants.py [hunk 3]
```diff
@@ -894,7 +895,7 @@
                   "Es freut mich, Sie kennenzulernen. Mein Name ist [Botname], und ich bin hier, um [Sie bei der Terminvereinbarung zu unterstützen]. Kannst du mir helfen, indem du mir deinen Namen nennst?",
                   "Dann sind Sie bei mir genau richtig. Ich kann Sie auf jeden Fall dabei unterstützen, die richtige Versicherung für Ihre Bedürfnisse zu finden! Aber zuerst würde ich Sie gerne besser kennenlernen. Wie heißt du?",
                   "Hallo, ich bin Ergo, der virtuelle Assistent von Godrej Properties. Ich bin hier, um Ihnen die besten Gewerbeflächen zu zeigen, die wir haben und die alle Ihre Anforderungen erfüllen. Darf ich Ihren Namen erfahren?"],
-    FRENCH_LANG: ["nom", "Présentez-vous", "Entrez votre nom", "Quel est votre nom", "A été réservé sous?",
+    FRENCH_LANG: ["votre nom","son nom", "Présentez-vous", "Entrez votre nom", "Quel est votre nom", "A été réservé sous?",
```
Comment (haptik-deployment) [inline ner_v1/detectors/textual/name/lang_constants.py:898]: E231 missing whitespace after ','
E501 line too long (123 > 119 characters)

**Comment by haptik-deployment** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Review by krupalmodi18 — APPROVED** [review]

**Pull request merged**